robottelo/ui/usergroup.py

```diff
@@ -59,31 +59,25 @@ def update(self, old_name, new_name=None,
             roles = []
         if new_roles is None:
             new_roles = []
-        element = self.search(old_name)
-
-        if element:
-            element.click()
-            self.wait_for_ajax()
-            if new_name:
-                if self.wait_until_element(locators['usergroups.name']):
-                    self.field_update('usergroups.name', new_name)
-            self.configure_entity(
-                users, FILTER['usergroup_user'], new_entity_list=new_users)
-            if roles or new_roles:
-                self.click(tab_locators['usergroups.tab_roles'])
-                if "admin" in roles or "admin" in new_roles:
-                    self.click(locators['usergroups.admin'])
-                else:
-                    self.configure_entity(
-                        entity_list=roles,
-                        new_entity_list=new_roles,
-                        filter_key=FILTER['usergroup_role'],
-                        entity_select=entity_select,
-                    )
-            if refresh_extusrgp:
-                self.click(tab_locators['usergroups.tab_external'])
-                self.click(locators['ldapserver.refresh'])
+        self.click(self.search(old_name))
+        if new_name:
+            if self.wait_until_element(locators['usergroups.name']):
+                self.field_update('usergroups.name', new_name)
+        self.configure_entity(
+            users, FILTER['usergroup_user'], new_entity_list=new_users)
+        if roles or new_roles:
+            self.click(tab_locators['usergroups.tab_roles'])
+            if "admin" in roles or "admin" in new_roles:
+                self.click(locators['usergroups.admin'])
             else:
-                self.click(common_locators['submit'])
+                self.configure_entity(
+                    entity_list=roles,
+                    new_entity_list=new_roles,
+                    filter_key=FILTER['usergroup_role'],
+                    entity_select=entity_select,
+                )
+        if refresh_extusrgp:
```

Comment:
1) Absolutely have no idea why we need that functionality, especially when I look into the code where that parameter is used:
```
            self.usergroup.update(
                self.usergroup_name,
                refresh_extusrgp=True,
            )
        with Session(
```
so we do logout right after refreshing something
2) Mentioned locator `ldapserver.refresh` doesn't exist on `usergroups.tab_external` sub-tab page
3) Functionality is blocked for one year by a defect https://bugzilla.redhat.com/show_bug.cgi?id=1221971

Gonna remove that code and back to it once test case is unblocked